Testing/SystemTests/tests/framework/CodeConventions.py

```diff
@@ -17,39 +17,43 @@
 
 # TODO this list should be empty
 ALG_BAD_PARAMS = {
-    "Bin2DPowderDiffraction(v1)": ("dSpaceBinning", "dPerpendicularBinning"),
-    "CalculateUMatrix(v1)": ("a", "b", "c", "alpha", "beta", "gamma"),
-    "ConvertToMD(v1)": ("dEAnalysisMode"),
-    "ConvertToMDMinMaxLocal(v1)": ("dEAnalysisMode"),
-    "ConvertToMDMinMaxGlobal(v1)": ("dEAnalysisMode"),
-    "EstimateDivergence(v1)": ("alpha", "beta0", "beta1"),
-    "FindUBUsingLatticeParameters(v1)": ("a", "b", "c", "alpha", "beta", "gamma"),
-    "IndexSXPeaks(v1)": ("a", "b", "c", "alpha", "beta", "gamma", "dTolerance"),
-    "LoadDNSSCD(v1)": ("a", "b", "c", "alpha", "beta", "gamma"),
-    "ModeratorTzero(v1)": ("tolTOF"),
-    "MuscatFunc(v1)": ("dQ", "dW"),
-    "OptimizeCrystalPlacement(v1)": ("nPeaks", "nParams", "nIndexed"),
-    "PDFFourierTransform(v1)": ("rho0"),
-    "PDFFourierTransform(v2)": ("rho0"),
-    "PoldiAutoCorrelation(v5)": ("wlenmin", "wlenmax"),
-    "PoldiLoadChopperSlits(v1)": ("nbLoadedSlits"),
-    "PoldiLoadSpectra(v1)": ("nbSpectraLoaded"),
-    "PoldiProjectRun(v1)": ("wlenmin", "wlenmax"),
-    "PoldiRemoveDeadWires(v1)": ("nbExcludedWires", "nbAuteDeadWires"),
-    "SaveIsawQvector(v1)": ("Qx_vector", "Qy_vector", "Qz_vector"),
-    "SCDCalibratePanels(v1)": ("a", "b", "c", "alpha", "beta", "gamma", "useL0", "usetimeOffset",
-                               "usePanelWidth", "usePanelHeight", "usePanelPosition",
-                               "usePanelOrientation", "tolerance", "MaxPositionChange_meters"),
+    "Bin2DPowderDiffraction(v1)":("dSpaceBinning","dPerpendicularBinning"),
+    "PowderReduceP2D(v1)":("dSpaceBinning","dPerpendicularBinning"),
```

Comment:
I think you only changed one line in this file right? If you could revert the previous formatting that would be great